webapi/src/it/scala/org/knora/webapi/core/LayersTest.scala

```diff
@@ -24,129 +30,74 @@ object LayersTest {
   type DefaultTestEnvironmentWithoutSipi = LayersLive.DspEnvironmentLive with FusekiTestContainer with TestClientService
   type DefaultTestEnvironmentWithSipi    = DefaultTestEnvironmentWithoutSipi with SipiTestContainer
 
-  /**
-   * All live layers with both Fuseki and Sipi testcontainers
-   */
-  val defaultLayersTestWithSipi =
-    ZLayer.make[DefaultTestEnvironmentWithSipi](
-      ActorSystem.layer,
+  type CommonR0 = ActorSystem with IIIFService with AppConfig
+  type CommonR = ApiRoutes
```

Comment:
What does the `CommonR0` and `CommonR` stand for? `R` is for "Routes", I guess? What about the `0`? Could you write this explicitly? It would be clearer, I guess.

Reply:
The names for the parameters come from the signature of the function which uses it:
`def makeSome[R0, R]`. I am open for suggestions for a better naming.